include/fe/fe_interface.h

```diff
@@ -31,242 +29,239 @@
 namespace libMesh
 {
 
-
-// forward declarations
-class BoundaryInfo;
-class DofConstraints;
-class DofMap;
-class Elem;
-class FEType;
-class FEComputeData;
-class Point;
-class MeshBase;
-enum FEFamily : int;
-enum Order : int;
-enum FEFieldType : int;
-enum ElemType : int;
-enum FEContinuity : int;
+  // forward declarations
+  class BoundaryInfo;
+  class DofConstraints;
+  class DofMap;
+  class Elem;
+  class FEType;
+  class FEComputeData;
+  class Point;
+  class MeshBase;
+  enum FEFamily : int;
+  enum Order : int;
+  enum FEFieldType : int;
+  enum ElemType : int;
+  enum FEContinuity : int;
 
 #ifdef LIBMESH_ENABLE_PERIODIC
-class PeriodicBoundaries;
```

Comment:
What fraction of the diff here is whitespace changes?  Those are fine if they're motivated and restricted to a separate commit (or ideally PR), but if they're mixed in to a commit with substantive changes then that's basically turning PR review (as well as any future git diffs that cross the commit boundary) into an Obfuscated C Code contest entry.

I'm guessing this was an automated editor thing?  I told vim a long time ago to automatically run clang-format whenever I saved a MOOSE file (or any file in a subdirectory of a directory with a `.clang-format` file), but soon afterward I was very unpleasantly surprised the first time I tried to save a libMesh file that I was working on from *within the MOOSE submodule*.

Reply:
You're correct in that this is an automated editor thing. I also noticed the drastic edits with using the auto editing in libMesh files. I thought I corrected those unwanted edits, but I missed this file. I will fix the unwanted whitespace edits.